README.md

```diff
@@ -3,50 +3,163 @@
+```
+
+We _strongly_ recommend using your system package manager to install Mise.
+
+Additional installation documentation can be found
+[here](https://mise.jdx.dev/getting-started.html#alternate-installation-methods)
+
+Once you have Mise installed, you will need to activate it in your shell.
+
+```sh
+# Bash
+echo 'eval "$(mise activate bash)"' >> ~/.bashrc && source ~/.bashrc
+
+# Zsh
+echo 'eval "$(mise activate zsh)"' >> ~/.zshrc && source ~/.zshrc
+
+# Fish
+echo 'mise activate fish | source' >> ~/.config/fish/config.fish && source ~/.config/fish/config.fish
+```
+
+Once Mise is installed and activated, you can run `mise trust` and then
+`mise install` to install all the required tools.
+
+Other shells are supported as well. Please refer to the
+[Mise documentation](https://mise.jdx.dev/getting-started.html#shells) for more
+information.
+
+> [!NOTE]
+> At the time of writing, Mise
+> [does not support Windows](https://mise.jdx.dev/faq.html#windows-support).
+> WSL is recommended for Windows users.
+>
+> If you are using WSL, you will need to take some
+> [additional steps](https://kind.sigs.k8s.io/docs/user/using-wsl2/) for Kind to
+> work.
 
 ## Docs
 
-You can find more detailed documentation in the [`./docs` folder](docs/README.md). The `Quick Start`, `Common Steps` and `Example Flows` will be most useful.
+You can find more detailed documentation in the
+[`./docs` folder](docs/README.md). The `Quick Start`, `Common Steps` and
+`Example Flows` will be most useful.
 
 ## Contributing
 
-While this project was primarily developed to meet the needs of [Yoma](https://yoma.africa), it is an open-source project that can be used for other Hyperledger Aries projects as well. We welcome contributions from everyone. Feel free to engage in discussions, create a PR against a ticket, or open an issue yourself, following the GitHub guidelines.
+While this project was primarily developed to meet the needs of
+[Yoma](https://yoma.world), it is an open-source project that can be used for
+other Hyperledger Aries projects as well. We welcome contributions from
+everyone. Feel free to engage in discussions, create a PR against a ticket, or
+open an issue yourself, following the GitHub guidelines.
 
 ## Running Tests
 
-The tests use `pytest`. For convenience, use GNU Make to run them. First, ensure the project is running locally by executing `make start`. In another shell, you can now run `make tests` to run all tests or `make unit-tests` to run only the unit tests.
-
-Without using `make`, you can run:
+The tests use `pytest`. For convenience, use Mise to run them. First, ensure
+the project is running locally by executing `mise run tilt:up`. In another shell,
+you can now run `mise run tests` to run all tests or `mise run tests:unit` to run
+only the unit tests.
 
 ```bash
-./manage up
+mise run tilt:up
 ```
 
 Install the requirements:
 
 ```bash
-bash scripts/install_all_dependencies.sh
+mise run poetry:install:all
 ```
 
```

Comment:
maybe add example.env with all the urls

Reply:
@cl0ete, please check latest commits and let me know your thoughts